src/dev-app/theme.scss

```diff
@@ -13,6 +14,9 @@
 @include angular-material-typography-mdc();
 @include mat-edit-typography(mat-typography-config());
 
+// Include base styles for strong focus indicators.
+@include mat-strong-focus-indicators();
```

Comment:
To me it feels a little weird to have this go through mixins, because the consumer would have to include at least two of them. Could we control it through a Sass variable instead? That way we can opt not to emit styles inside the component's styles and resolve the issue I mentioned below.

Reply:
We could change it so that they only need to include one, `mat-strong-focus-indicators` that both turns on the focus indicators and sets the color.

Do you have an example of how we can control it via a Sass variable?

Reply:
It would have to look something like this:

variables.scss
```
$strong-focus-indication: false !global;
```

consumer-styles.scss
```
$strong-focus-indication: true !global;
```

some-component.scss
```
@import '../variables';

@if $strong-focus-indication {
  .mat-focus-indicator {}
}
```

AFAIK this has a dependency on the import order of files though. What do you think @jelbourn?

Reply:
I'm generally opposed to the idea of APIs that rely on overriding Sass variables vs. passing arguments to mixins/functions. (I've actually authored Google's internal Sass style guide to ban this).

The benefit of having two mixins is that people can change the color of the focus indicator without duplicating the main indicator styles, e.g.
```scss
@include mat-strong-focus-indicators();

// Default indicator color
@include mat-strong-focus-indicators-theme($theme);

.element-with-dark-background {
  @include mat-strong-focus-indicators-theme($alternate-color);
}
```